util/logging-utils/src/main/java/org/fidoalliance/fdo/loggingutils/LoggerService.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Intel Corporation
+// SPDX-License-Identifier: Apache 2.0
+
+package org.fidoalliance.fdo.loggingutils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Provides a wrapper class for Slf4j Logger and supporting centralized infrastructure for logging.
+ */
+public class LoggerService {
+
+  public static Logger logger;
+
+  public LoggerService(Class<?> className) {
+    logger = LoggerFactory.getLogger(className);
+  }
+
+  public void info(Object log) {
+    logger.info(log.toString());
+  }
+
+  public void debug(Object log) {
+    logger.debug(log.toString());
+  }
+
+  public void error(Object log) {
+    logger.error(log.toString());
+  }
+
+  public void warn(Object log) {
+    logger.warn(log.toString());
+  }
+
+}
```

Comment:
I don't understand why we're wrapping SLF4J instead of using it directly.

Reply:
This is taken up based on earlier experience with RV hosting, where having a separate logging service helps to easily switch to a different logging mechanism (say non-SLF4J based) based on need.